src/features/applicationList/__tests__/utils.test.ts

```diff
@@ -0,0 +1,106 @@
+import { getBerthApplicationData } from '../utils';
+import { mockData } from '../__fixtures__/mockData';
+import {
+  BERTH_APPLICATIONS,
+  BERTH_APPLICATIONS_berthApplications as BerthApplicationNodeConnection,
+  BERTH_APPLICATIONS_berthApplications_edges as BerthApplicationNodeEdge,
+} from '../__generated__/BERTH_APPLICATIONS';
+import { ApplicationStatus } from '../../../@types/__generated__/globalTypes';
+
+const mockDataApplications = (mockData.berthApplications as BerthApplicationNodeConnection).edges;
+
+const buildTestData = (edges: (BerthApplicationNodeEdge | null)[]): BERTH_APPLICATIONS => ({
+  berthApplications: {
+    __typename: 'BerthApplicationNodeConnection',
+    count: 1,
+    edges: edges,
+  },
+  boatTypes: [{ __typename: 'BoatTypeType', id: '1', name: 'Purjevene' }],
+});
+
+describe('utils', () => {
+  describe('getBerthApplicationData', () => {
+    it('should return berth application data', () => {
+      expect(getBerthApplicationData(mockData)).toMatchSnapshot();
+    });
+
+    it('should use defaults for nullable fields', () => {
+      const testEdge: BerthApplicationNodeEdge = {
```

Comment:
Could we move this to `__fixtures__/mockData.ts`?

Reply:
Yes. I included it into the set of mock data and changed some field values